### PR TITLE
Update docs of test key in writing E2E

### PR DESCRIPTION
**Pull request by saturninoabril**
Inputs from - https://github.com/mattermost/mattermost-server/issues/9805#issuecomment-528063746

Screenshot (updated per latest comment):
Screen Shot 2019-09-06 at 11 07 44 PM

### CODE REVIEW

File: site/content/contribute/webapp/end-to-end-tests.md [hunk 1]
```diff
@@ -81,23 +81,27 @@ Inside of the `integration` directory, there are additional directories that bre
...
         }
     }
     ```
 
-    Where `"M14014"` is the test key and `"Recently used emojis are shown 1st"` is the test description.  This information is stated in Github's [help-wanted ticket](https://github.com/mattermost/mattermost-server/issues/10246) and [Jira ticket](https://mattermost.atlassian.net/browse/MM-14014).
+    Where `"M15011"` is the key and `"Emojis show as jumbo in reply thread"` is the test description.
+
+    The naming convention used is the name of the tab on the release testing spreadsheet e.g. AS = Account Settings; M = Messaging, etc. and the numerical value is from Jira ticket identifier. So `"M15011"` means "Messaging" on release testing spreadsheet with `MM-15011` Jira ticket.
+
+    For those writing E2E from help-wanted tickets with `Add E2E Tests` label, the `Test Key` is available in Github [issue itself](https://github.com/mattermost/mattermost-server/issues/10574).
```
Comment (jasonblais) [inline site/content/contribute/webapp/end-to-end-tests.md]:
```suggestion
    For those writing E2E from help-wanted tickets with `Add E2E Tests` label, the `Test Key` is available in the [GitHub issue itself](https://github.com/mattermost/mattermost-server/issues/10574).
```

Comment (saturninoabril) [reply site/content/contribute/webapp/end-to-end-tests.md]: Thanks Jason, added the change.

File: site/content/contribute/webapp/end-to-end-tests.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ Inside of the `integration` directory, there are additional directories that bre
 
     The naming convention used is the name of the tab on the release testing spreadsheet e.g. AS = Account Settings; M = Messaging, etc. and the numerical value is from Jira ticket identifier. So `"M15011"` means "Messaging" on release testing spreadsheet with `MM-15011` Jira ticket.
 
-    For those writing E2E from help-wanted tickets with `Add E2E Tests` label, the `Test Key` is available in Github [issue itself](https://github.com/mattermost/mattermost-server/issues/10574).
+    For those writing E2E from help-wanted tickets with `Add E2E Tests` label, the `Test Key` is available in the [GitHub issue itself](https://github.com/mattermost/mattermost-server/issues/10574).
```
Comment (abdusabri) [inline site/content/contribute/webapp/end-to-end-tests.md]: Thanks a lot @saturninoabril !

The label should be `Area/E2E Tests`, right?

Comment (saturninoabril) [reply site/content/contribute/webapp/end-to-end-tests.md]: Oh, thanks for catching that!  Now updated. (Also I've updated the screenshot in the description)

**Pull request merged**